modules/social_features/social_follow_user/social_follow_user.module

```diff
@@ -187,3 +211,90 @@ function social_follow_user_preprocess_flag(array &$variables): void {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter() for user_form().
+ */
+function social_follow_user_form_user_form_alter(array &$form, FormStateInterface $form_state): void {
+  $profile = _social_profile_get_profile_from_route();
+  $active = \Drupal::config('social_follow_user.settings')->get('status');
+
+  if ($profile instanceof ProfileInterface && $active) {
+    $following_enabled = \Drupal::service('social_follow_user.helper')
+      ->getFollowingStatus((int) $profile->get('uid')->target_id);
+
+    $form['profile_privacy']['social_profile_following_enabled'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Allow members to follow me'),
+      '#default_value' => $following_enabled,
+      '#description' => t('This section allows you to turn off the ability for 
```

Comment:
@volodymyr-sydor Please change to: _This selection allows you to turn **on** the ability for members to follow your activity and receive notifications about new content you create._

Reply:
done